source/features/comments-time-machine-links.tsx

```diff
@@ -15,6 +15,10 @@ import GetCommitAtDate from './comments-time-machine-links.gql';
 import {expectToken} from '../github-helpers/github-token.js';
 import getDefaultBranch from '../github-helpers/get-default-branch.js';
 
+const reactIssueComment = 'div.react-issue-comment'; // First comment
+const reactIssueBody = 'div.react-issue-body'; // Comments
+const reactCommitView = '[data-testid="review-thread"] > div';
+
```

Comment:
Can be merged into a [].join() string. It can then be used as `:is(${the var}) relative-date[the attribute]`